accepted/wasmtime-lts.md

```diff
@@ -0,0 +1,170 @@
+    30.0.0           :a1, 2025-02-20, 600d
+    31.0.0           :a2, 2025-03-20, 60d
+    32.0.0           :a2, 2025-04-20, 60d
+    33.0.0           :a2, 2025-05-20, 60d
+    34.0.0           :a2, 2025-06-20, 60d
+    35.0.0           :a2, 2025-07-20, 60d
+    36.0.0           :a2, 2025-08-20, 60d
+    37.0.0           :a2, 2025-09-20, 60d
+    38.0.0           :a2, 2025-10-20, 60d
+    39.0.0           :a2, 2025-11-20, 60d
+    section lts2
+    40.0.0           :a1, 2025-12-20, 600d
+    41.0.0           :a2, 2026-01-20, 60d
+    42.0.0           :a2, 2026-02-20, 60d
+    43.0.0           :a2, 2026-03-20, 60d
+    44.0.0           :a2, 2026-04-20, 60d
+    45.0.0           :a2, 2026-05-20, 60d
+    46.0.0           :a2, 2026-06-20, 60d
+    47.0.0           :a2, 2026-07-20, 60d
+    48.0.0           :a2, 2026-08-20, 60d
+    49.0.0           :a2, 2026-09-20, 60d
+    section lts3
+    50.0.0           :a1, 2026-10-20, 600d
+```
+
+This means that once two LTS channels are in play there will be anywhere from 3
+to 4 releases of Wasmtime which receive security fixes at any one point in
+time.
+
+[patches]: https://docs.wasmtime.dev/contributing-release-process.html#releasing-a-patch-version
+
+# Rationale and alternatives
+[rationale-and-alternatives]: #rationale-and-alternatives
+
+* The major alternative to this design is to not support an LTS at all and
+  continue on Wasmtime's current path of only guaranteeing the
+  current-and-previous releases get security fixes. It's seen though that this
+  is likely excluding too many potential users of Wasmtime, hence this RFC.
+
+* Another alternative is to support only one LTS at a time. The problem with
+  this is that users of an LTS should have a period of time to upgrade from one
+  to the other where both LTS releases are supported. Otherwise if an upgrade
+  takes an unexpectedly long time then if a security issue happens it won't get
+  backported to the old LTS release.
+
+* There's always the possibility of supporting more LTS releases or supporting
+  them for longer. The two-release and ten-versions-apart are relatively
+  arbitrary at this time. Putting LTS releases 10 versions apart just makes them
+  easy to identify (30, 40, 50, ...). While some overlap is required (e.g at
+  least 11 months of support) it's not necessarily required to have 20 months of
+  support. For now though it's predicted this won't be too onerous to support.
+
+# Open questions
+[open-questions]: #open-questions
+
+- What is the exact process by which release automation does a "dry run" every
+  so often? And how frequently is "every so often"?
+
+- Supporting a release for 20 versions is, as of now, probably a little
+  ambitious. Should we include wording saying that this is sort of a "trial run"
+  and we reserve the right to decrease the support window? Should we start with
+  something smaller like supporting for 12 months first and then reevaluate in
+  the future if it's worth widening to 20?
```

Comment:
I would be in favor of starting with 15 months of support (2-month overlap seems perhaps too short; half of time till the next LTS seems nice and round) and then re-evaluating whether to extend that to 20 after the first LTS goes out of support, as you suggest. (FWIW: At that time, if we extend to 20, I don't think we should need to go through the whole RFC process again.)

Reply:
Sounds good to me, updated 👍